### PR TITLE
Remove automapper for list prices

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/CataloguePrice.cs [hunk 1]
```diff
@@ -40,5 +41,12 @@ public override string ToString()
         {
             return $"£{Price.Value:F} {PricingUnit?.Description}".Trim();
         }
+
+        public string PriceString()
+        {
+            return $"£{Price.Value:F}";
+        }
+
+        public string PricingUnitString() => $"{PricingUnit.Description} {(TimeUnit.HasValue ? TimeUnit.Value.Description() : string.Empty)}".Trim();
```
Comment (steves-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.EntityFramework/Catalogue/Models/CataloguePrice.cs:50]: PricingUnitString will be terminated with a space if there is no description.

**Review by steves-bjss — DISMISSED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/ListPriceModel.cs [hunk 2]
```diff
@@ -1,13 +1,29 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using NHSD.GPIT.BuyingCatalogue.EntityFramework.Catalogue.Models;
 
 namespace NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Solutions.Models
 {
     public class ListPriceModel : SolutionDisplayBaseModel
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/ListPriceModel.cs:7]: Can we seal this now?

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/ListPriceModel.cs [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
     {
+        public ListPriceModel()
+        {
+
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/ListPriceModel.cs:11]: Should be no white space between the braces here.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/PriceViewModel.cs [hunk 4]
```diff
@@ -1,11 +1,19 @@
-﻿namespace NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Solutions.Models
+﻿using NHSD.GPIT.BuyingCatalogue.EntityFramework.Catalogue.Models;
+
+namespace NHSD.GPIT.BuyingCatalogue.WebApp.Areas.Solutions.Models
 {
     public class PriceViewModel
```
Comment (bjss-chris) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Models/PriceViewModel.cs:5]: Can we seal this now?

**Comment by PhysicsJon** [comment]
Closing due to me messing up merge conflicts in a poor way

**Pull request closed without merging**